airbyte-integrations/connectors/destination-postgres/gradle.properties

```diff
@@ -1,3 +1,5 @@
 # our testcontainer has issues with too much concurrency.
 # 4 threads seems to be the sweet spot.
 testExecutionConcurrency=4
+# large sync test takes a while, add 15m timeout.
+JunitMethodExecutionTimeout=15 m
```

Comment:
please don't do a blanket execution timeout bump. You can use the @Timeout on a single function to do the same thing